cmd/shipperctl/cmd/backup/prepare.go

```diff
@@ -0,0 +1,86 @@
+package backup
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/bookingcom/shipper/cmd/shipperctl/configurator"
+	"github.com/bookingcom/shipper/cmd/shipperctl/release"
+)
+
+var (
+	prepareBackupCmd = &cobra.Command{
+		Use:   "prepare",
+		Short: "Get yaml of application and release objects prepared for backup",
+		Long: "Get yaml of application and release objects prepared for backup" +
+			"where every application will be bundled with it's releases",
+		RunE: runPrepareCommand,
+	}
+)
+
+func init() {
+	BackupCmd.AddCommand(prepareBackupCmd)
+}
+
+func runPrepareCommand(cmd *cobra.Command, args []string) error {
+	kubeClient, err := configurator.NewKubeClientFromKubeConfig(kubeConfigFile, managementClusterContext)
+	if err != nil {
+		return err
+	}
+
+	shipperClient, err := configurator.NewShipperClientFromKubeConfig(kubeConfigFile, managementClusterContext)
+	if err != nil {
+		return err
+	}
+
+	namespaceList, err := kubeClient.CoreV1().Namespaces().List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	var errList []string
+
+	releasesPerApplications := []releasesPerApplication{}
```

Comment:
nit: semantically, this feels like it should be a map, it won't change much the code itself, but it might be slightly reduce the cognitively complexity. That is probably just a personal preference.